realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -175,6 +192,16 @@ public boolean isInTransaction() {
         return nativeIsInTransaction(nativePtr);
     }
 
+    public void setSchemaVersion(long schemaVersion) {
+        // FIXME migrate to ObjectStore
```

Comment:
There is no easy way to set it. That is, you will have to implement it in Realm Object Store first.